### ISSUE TITLE
Images won't load when data connection breaks during gameplay

### ISSUE THREAD

**Issue by doctor-g** [spring2017gamestudio/prairie-creek#119]
The whole game logic is downloaded at the beginning of play, but as you walk around the park, you might drop the data connection. Then, when discovering a spirit, there's no image shown---because, of course, there's no data connection. This could be avoided by prefetching all the images at the start of the game, when there must be a data connection.

**Comment by doctor-g** [comment]
I'm upgrading this to a bug since it breaks the fundamental feedback system as it is.

**Comment by doctor-g** [comment]
Oops, that was an accidental closing above, but I don't think there's a way to undo that. Sorry :)

**Comment by doctor-g** [comment]
I started fiddling with this during a spare hour, and I have a path to try. I pushed a branch `image-cache` with some proof-of-concept code, but it's far from ready; I just had to push it to sync between devices.

The current running idea is to make an `image-cache` element that preloads all of the images and puts them into an associative array. This array can then be sent to a new `spirit-image` element that reads from the cache instead of manually setting the `img src`. I am not sure that this will work, since it took me most of my time just to come up with the idea, but I hope to fiddle with it some more. If someone else has a better idea and can beat me to it, go for it!
